de/nmichael/efa/data/storage/DataFile.java

```diff
@@ -241,8 +242,10 @@ public synchronized void openStorageObject() throws EfaException {
             if (recovered || shouldWriteMirrorFile()) {
                 saveStorageObject();
             }
-            if ((Daten.project != null) && (Daten.project.getProjectStorageType() == IDataAccess.TYPE_EFA_CLOUD))
-                Ecrid.addAll(this.getPersistence());
+            String fName = filename.substring(filename.lastIndexOf('.') + 1);
+            if (TableBuilder.tablenames.contains(fName))
+                if (Daten.project.getProjectStorageType() == IDataAccess.TYPE_EFA_CLOUD)
```

Comment:
Check for Daten.project != null has been discarded. Why?
Very often, Daten.project CAN actually be null:
- directly after installing efa with no project open
- closing projects during restores with config data
- closing / reopening projects when loading a diffrent project
I suggest to re-introduce the check to daten.project != null.